website/content/en/docs/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -77,7 +77,14 @@ The following cluster configuration will:
 * Create a role to allow spot instances.
 * Run helm to install karpenter
 
+{{#tabs name:"create-cluster" tabs:"With NodeGroup,With Fargate"}}
```

Comment:
When I look at the preview that's deployed from this PR, I don't see the tabs getting rendered. Are we missing a short-code here with hugo/docsy that will give us the tabs?

Reply:
I have fixed this issue in the latest commit. Pls check.